addons/mail/static/src/views/fields/emojis_char_field/emojis_char_field.xml

```diff
@@ -2,11 +2,19 @@
 
 <templates xml:space="preserve">
     <t t-name="mail.EmojisCharField" t-inherit="web.CharField" t-inherit-mode="primary" owl="1">
+        <xpath expr="//input" position="attributes">
+            <attribute name="t-on-input">onInput</attribute>
+        </xpath>
         <xpath expr="//span[1]" position="attributes">
             <attribute name="t-ref">targetReadonlyElement</attribute>
         </xpath>
-        <xpath expr="/*[last()]/*[last()]" position="after">
-            <EmojisDropdown onEmojiClick="onEmojiClick" readonly="props.readonly" type="'char'"/>
+        <xpath expr="//input" position="after">
+                <button
+                    class="btn"
+                    t-ref="emojisButton"
+                    t-on-click="onClickEmojiButton">
```

Comment:
can we do something like this directly ? (so we don't need to create a wrapper method in `EmojisTextField` just for that)
```suggestion
                    t-on-click="emojiTextField.onClickButtonEmojis">
```